### PR TITLE
Run PHPCS on PHP 8

### PR DESCRIPTION
**Pull request by derrabus**

### CODE REVIEW

File: .github/workflows/coding-standards.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.4"
+          - "8.0"
```
Comment (ostrolucky) [inline .github/workflows/coding-standards.yml:20]: Perhaps we should run it with 7.1, that's the compatibility we need to maintain.

Comment (derrabus) [reply .github/workflows/coding-standards.yml:20]: I've added 7.1 as well.

Comment (greg0ire) [reply .github/workflows/coding-standards.yml:20]: :thinking: I think one run should be enough, if we add `<config name="php_version" value="70100"/>` to the configuration.

**Review by ostrolucky — APPROVED** [review]

File: .github/workflows/coding-standards.yml (hunk 1 above)
Comment (derrabus) [reply .github/workflows/coding-standards.yml:20]: Let's check that. 🙂

**Review by SenseException — APPROVED** [review]

File: .github/workflows/coding-standards.yml (hunk 1 above)
Comment (derrabus) [reply .github/workflows/coding-standards.yml:20]: Yes, that does the trick. I could even remove two more rules from the exclude list now. 🎉

**Pull request merged**